homeassistant/components/netgear/device_tracker.py

```diff
@@ -110,6 +110,7 @@ def scan_devices(self):
                     or dev.name in self.excluded_devices
                 )
             )
+            tracked = tracked and dev.link_rate is not None
```

Comment:
`tracked` keeps track if a device is tracked. `link_rate` should not impact the value of this variable.

Reply:
In this case, "tracked" is a boolean that determines if the device should be reported as seen or not. self.last_results contains all the seen devices. However, pynetgear returns devices even after they've disconnected from WiFi, however, for these disconnected devices, link_rate is None. 

Unless you just mean that you want link_rate to not contaminate the value of tracked, in which case, I can change it to a simple "if dev.link_rate is None" inside of the "if tracked:" block.

Reply:
Yeah, we should not contaminate the value. Also add a comment why `link_rate is None` is checked.

Reply:
I have made the requested changes.